addons/dark-www/experimental_scratchwww.css

```diff
@@ -24,7 +24,7 @@ a.social-messages-profile-link,
 .social-form,
 body:not(.sa-body-editor) .join-flow-title,
 .initiatives-section .initiatives-tools .subsection-tag.collaborator {
-  color: #fff;
+  color: #fff !important;
```

Comment:
Add `.semicolon` to the list of selectors (after `#donor`) instead of `!important` (which would likely break things).